main.cpp

```diff
@@ -126,6 +128,12 @@ void parseArgs(int argc, char **argv, Settings &settings) {
             case 'i':
                 settings.info = true;
                 break;
+            case 'f':
+                {
+                    ifstream ifs(optarg);
```

Comment:
Please test `ifs` and error/exit if not found.

Something simple like "config file /path/to/.xlayoutdisplay not found, exiting"